source/FreeRTOS_UDP_IPv6.c

```diff
@@ -613,6 +613,10 @@ BaseType_t xProcessReceivedUDPPacket_IPv6( NetworkBufferDescriptor_t * pxNetwork
             #endif /* ipconfigUSE_NBNS */
             {
                 xReturn = pdFAIL;
+
+                #if ( ipconfigREPLY_ICMP_PORT_UNREACHABLE != 0 )
+                    FreeRTOS_SendDestUnreachableIPv6( ipICMP_DEST_UNREACH_PORT_UNREACHABLE, pxNetworkBuffer );
```

Comment:
Just curious, should we reply "ICMP port unreachable" to any type of target IP-address?
See also function `xIPv6_GetIPType()`.